tests/shtest

```diff
@@ -103,28 +103,44 @@ if [ $n -ne $nref ]; then
   exit 1
 fi
 
+## Test parse error message
+echo 123 > $d/input_valid
+echo 456 > $d/input_error_line2
+echo asd >> $d/input_error_line2
+
+cat > $d/expected_out <<EOF
+123
+456
+EOF
+cat > $d/expected_err <<EOF
+jq: parse error (at $d/input_error_line2:2): Invalid numeric literal at line 4, column 0
```

Comment:
`line 4` here is a bit confusing, i guess that is counting lines from all input files seen so far